admin/openwisp-config/Makefile

```diff
@@ -38,23 +38,24 @@ endef
 
 define Package/openwisp-config/conffiles
 /etc/config/openwisp
+/etc/openwisp/
```

Comment:
Would be great to mention this additional thing in the commit description.

Reply:
It was added by @pandafy, is it ok? Any other blocker?